docs/guides/node-operator-manual.md

```diff
@@ -178,30 +214,44 @@ Etherscan pages for the Görli/Prater contracts:
 - [`Lido`](https://goerli.etherscan.io/address/0x1643E812aE58766192Cf7D2Cf9567dF2C37e9B7F#readProxyContract)
 - [`NodeOperatorsRegistry`](https://goerli.etherscan.io/address/0x9D4AF1Ee19Dad8857db3a45B0374c81c8A1C6320)
 
+Etherscan pages for the Mainnet contracts:
+
+- [`Lido`](https://etherscan.io/address/0xae7ab96520de3a18e5e111b5eaab095312d7fe84#readProxyContract)
+- [`NodeOperatorsRegistry`](https://etherscan.io/address/0x55032650b14df07b85bf18a3a3ec8e0af2e028d5#readProxyContract)
+
 [`addsigningkeysoperatorbh` function]: https://github.com/lidofinance/lido-dao/blob/971ac8f/contracts/0.4.24/nos/NodeOperatorsRegistry.sol#L250
 [`getoperators()` function]: https://github.com/lidofinance/lido-dao/blob/971ac8f/contracts/0.4.24/Lido.sol#L361
 [`lib/abi/nodeoperatorsregistry.json`]: https://github.com/lidofinance/lido-dao/blob/971ac8f/lib/abi/NodeOperatorsRegistry.json
 [`nodeoperatorsregistry.getnodeoperator`]: https://github.com/lidofinance/lido-dao/blob/971ac8f/contracts/0.4.24/nos/NodeOperatorsRegistry.sol#L335
 
-#### Using the key submitter UI
+#### Using the batch key submitter UI
+
+Lido has specialised key submitters: [Mainnet web interface for submitting the keys] and a [Testnet web interface for submitting the keys].
 
-Lido has the specialized [web interface for submitting the keys].
+It will require that all keys are validated first - click the check button. Please be patient as validation can take a long time. Also, please don't update the page - UI has a built-in request failure retries integrated and will display an error if all request fail.
```

Comment:
1. What is the ballpark of a long time here? It's about 10 minutes for 100 keys, right?
2. `built-in request failure retries integrated` — that's a bit hard to parse. Seems like something like "the page repeats the request if the current check has failed, and will show an error after couple of retries" would be a bit easier to read.

Reply:
1. No, much faster, but honestly we currently don't have even an idea of the processing time after our checkers will be scaled - better wait and see. We can add something vague though - eg 1k keys under a minute, but it doesn't at all account for simultaneous checks - we don't have a queue implemented
2. Agree, will change the wording